parsons/databases/redshift/redshift.py

```diff
@@ -488,6 +488,8 @@ def copy(
         alter_table_cascade=False,
         aws_access_key_id=None,
         aws_secret_access_key=None,
+        aws_session_token=None,
+        use_env_token=False,
```

Comment:
I looked through the code, and I don't _think_ that this is being used. It gets passed around a bit, but ultimately `temp_s3_copy` is passing the field from the Redshift instance, not the parameters added in this PR, into the `S3` object. See rs_copy_table.py:L190.

And I agree with the way that it's technically implemented; the better design here is to defer to the `use_env_token` from the Redshift constructor. I think you can just remove the `use_env_token` parameter from the `copy` method, and the other methods where it's just being passed around.